service/controller/v11/resource/pod/delete.go

```diff
@@ -196,12 +196,14 @@ func (r *Resource) createNodeConfig(ctx context.Context, pod *corev1.Pod) error
 	}
 
 	_, err = r.g8sClient.CoreV1alpha1().NodeConfigs(n).Create(c)
-	if err != nil {
+	if apierrors.IsAlreadyExists(err) {
+		r.logger.LogCtx(ctx, "level", "warning", "message", "node config for guest cluster node already exists")
```

Comment:
I wonder if this is really `warning`. Should be alerted by this log?

Reply:
I am leaning towards debug as well.

Reply:
By reading the code, this should never happen, but we did. We can handle this gracefully, but I see it as a warning sign that something is not necessarily right. Not a pure error as everything is fine despite of this, but still abnormal.

Reply:
The thing is we can't really react on this warning. This would be a good place to apply "ops-recipe" as we discussed some time ago in the errors approach issue. Maybe it makes more sense to add such a field to our logging.

Reply:
Created TODO to track this: https://github.com/giantswarm/giantswarm/issues/4325